march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -113,6 +114,7 @@ class MarchHardwareInterface : public hardware_interface::RobotHW
 
   std::vector<double> joint_effort_;
   std::vector<double> joint_effort_command_;
+  std::vector<double> joint_effort_command_copy_;
```

Comment:
I would call this vector `joint_last_effort_command_` since that better describes what it contains.